src/main/java/frc/robot/subsystems/vision/VisionConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.subsystems.vision;
+
+public class VisionConstants {
```

Comment:
This class is inside the `vision` package, so calling it `Constants` is enough. Don't forget the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.